packages/pds-ember/addon/components/pds/external-link/index.hbs

```diff
@@ -0,0 +1,15 @@
+<a
+  class="pds--external"
```

Comment:
Using a modifier class instead of an element class, because there really shouldn't be any difference in styling from typical links.  However, the modifier is in place in the event that it is needed later.